x-pack/plugins/secops/server/lib/uncommon_processes/elasticsearch_adapter.ts

```diff
@@ -77,7 +76,7 @@ export const formatUncommonProcessesData = (
   fields.reduce(
     (flattenedFields, fieldName) => {
       flattenedFields.uncommonProcess._id = hit._id;
-      flattenedFields.uncommonProcess.instances = hit.total;
+      flattenedFields.uncommonProcess.instances = getOr(0, 'total.value', hit);
```

Comment:
This is an interesting change. It is now `hit.total.value` rather than `hit.total` and it can be `undefined` as well?

Reply:
If so, that's fine, wondering if we can just directly use `hit.total.value` or if we have to use the `getOr`. I am fine with it if we have to use the `getOr` though.

Reply:
Yes that was just an easy fix, I also want to investigate the relation attribute to make sure that we do the right thing here and for other query in the future